operator/config/rbac/api/role.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: forklift-api
+rules:
+  - apiGroups:
+      - forklift.konveyor.io
+    resources:
+      - providers
+      - storagemaps
+    verbs:
+      - get
```

Comment:
also missing 'list' here, no?

Reply:
i don't think it's required. but sure, i will add it